README.md

```diff
@@ -9,45 +9,176 @@ Please create a branch called `homework`. Then, create a file named `homework.js
   <br>
     <code>summation(5) // should return 15 because 1+2+3+4+5=15</code>
   </li>
+//My answer:
+
+const sum = function(n){
+  let a =0 ;
+for(let i = 1;i <= n ; i++){
+	a=a+i
+}
+return a;
+}
+console.log(sum(5));
+---------------------------------------------------------
   <li>Create a program to get the sum of all the even numbers in a group
   <br>
     <code > summationEven(5) // should return 6 because 2+4=6</code>
   </li>
-    
+//My answer:
+const susummationEven = function(n){
+  let a = 0 ;
+
+  for(let i = 0 ; i <= n ; i++ ){
+  if(i%2 ==0)
+{
+  a = a + i;	
+} 
+  }
+   return a;
+}
+console.log(susummationEven(5))
+---------------------------------------------------------
   <li>Create a function to get the average of a group of numbers 
   <br>
     <code> avg([8, 2, 2, 4]) // should return 4</code>
   </li>
+//My answer:
+
+const average = function(avg){
+let sum=0;
+let av = 0;
+  for(let i = 0 ; i < avg.length; i++ ){
+     sum=sum + avg[i];
+  }
+  av=sum/(avg.length)
+ console.log(av);
+}
+const avg=[8, 2, 2, 4];
+average(avg)
+---------------------------------------------------------
   <li>Create a function to reverse the letters in a word
   <br>
     <code > reverse("caterpillar") // should return "rallipretac"</code>
   </li>
+//My answer:
+
+const rev =function(text){
+text = text.split("")
+text = text.reverse()
+text = text.join("")
+return text
```

Comment:
You can use all of these function together, like this: ` return text.split("").reverse().join("");`